collection-scripts/gather_running_vms

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+source $(dirname "$0")/common.sh
+BASE_COLLECTION_PATH="must-gather"
+get_operator_ns "kubevirt-hyperconverged"
+OPENSHIFT_CNV_PATH="${BASE_COLLECTION_PATH}/namespaces/${operator_ns}"
+
+function collect_running_vms_count() {
+  mkdir -p "${OPENSHIFT_CNV_PATH}"
+  oc get vmi --all-namespaces --no-headers | wc -l > "${OPENSHIFT_CNV_PATH}/running_vms_count.txt"
```

Comment:
@nunnatsa, @sradco Do you think it's misleading to put this under `openshift-cnv` dir? 
As it's gathered from `--all-namespaces`. 
We can put it under `cluster-scoped-resources` dir instead, but that's also misleading IMO, since VMs are not cluster scoped.

Reply:
I know that we collect and report a metric named `cnv:vmi_status_running:count`  about currently-running VMs. Can we pull it instead of listing all vmis on the must-gather pod and counting them?

Reply:
@sradco mentioned that we can't assume Prometheus is installed and available, and in order to get `cnv:vmi_status_running:count` value, we need to query Prometheus, so I'm not sure it's possible.

Reply:
@dankenigsberg @assafad  Actually, We can try to collect this from Prometheus, but I think using the oc command would be more reliable. OCP MG does have a a collection for Prometheus alerts (That I would like to add to CNV MG) and we can try to get the metric from it, But it can potentially be unreachable for different reasons.
Do you think it would be "good enough"? What would be the advantage?

Reply:
> Do you think it would be "good enough"? What would be the advantage?

The code we are adding here would be executed by every OpenShift customer looking for support, if they have Virt installed or not. I would like to ensure we burden them as little as possible. I suspected that collecting a single metric would be faster, simpler and more reliable than listing all VMs. I accept your verdict if this is not the case.

Reply:
Just mentioning that the command is skipped when Virt is not installed, using:
 https://github.com/openshift/must-gather/blob/197b7c6051deca66c76ecddc69c6b726d0d0f092/collection-scripts/common.sh#L11